.github/workflows/native_s3_dlr.yml

```diff
@@ -46,5 +38,50 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-2
       - name: Upload native DJL to S3
-        run: aws s3 cp *dlr.* s3://djl-ai/publish/dlr-${{ env.DLR_VERSION }}/native/${{ matrix.os }}/cpu/
+        run: aws s3 cp *dlr.* s3://djl-ai/publish/dlr-${{ env.DLR_VERSION }}/native/osx/cpu/
+        working-directory: neo-ai-dlr/build/lib
+
+  build-linux:
+    runs-on: ubuntu-latest
+    container: amazonlinux:2
+    steps:
+      - name: Install Environment
+        run: |
```

Comment:
If the centOS7 config is generic: https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action you can do this

Reply:
We need to create a new github repo and maintain the action. It's overkill.